backend/dal/Helpers/Extensions/ProjectExtensions.cs

```diff
@@ -348,7 +349,10 @@ public static IEnumerable<Entities.Parcel> GetSubdivisionParentParcels(this Enti
         /// <returns></returns>
         public static void TransferProjectProperties(this PimsContext context, Entity.Project originalProject, Entity.Project project)
         {
-            originalProject.Properties.ForEach(p =>
+            var parentParcels = originalProject.GetSubdivisionParentParcels();
+            context.DisposeSubdivisionParentParcels(parentParcels);
+            var propertiesWithNoSubdivisions = originalProject.Properties.Where(p => !parentParcels.Any(pp => p.Parcel?.Id == pp.Id));
+            propertiesWithNoSubdivisions.ForEach(p =>
```

Comment:
as stated in the description of the GRE story, parent agencies of a subdivision project shouldn't be transferred to the new agency.